padle/padle.py

```diff
@@ -0,0 +1,801 @@
+                MonsterImage.icon(m.monster_id))).to_embed()
+        await ctx.send(embed=m_embed)
+
+    async def get_past_padle_embed(self, ctx, day: int):
+        dbcog = await self.get_dbcog()
+        daily_scores = await self.config.save_daily_scores()
+        info = daily_scores[day - 1]
+        stats = info[1]
+        giveups = 0
+        completes = 0
+        average = 0
+        for item in stats:
+            if item == "X":
+                giveups += 1
+            else:
+                completes += 1
+                average += int(item)
+        guess_monster = dbcog.get_monster(int(info[0]))
+        embed = EmbedView(
+            EmbedMain(
+                title="PADle #{} Stats".format(day)),
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(guess_monster.monster_id))).to_embed()
+        if completes == 0:
+            embed.description = ("**" + MonsterHeader.menu_title(guess_monster).to_markdown() + "**\n" +
+                                 "**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: 0%\n"
+                                 "**Average Guess Count**: 0").format(completes, giveups)
+        else:
+            embed.description = ("**" + MonsterHeader.menu_title(guess_monster).to_markdown() + "**\n" +
+                                 "**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: {:.2%}\n"
+                                 "**Average Guess Count**: {:.2f}").format(completes, giveups,
+                                                                           completes / (completes + giveups),
+                                                                           (average / completes))
+        return embed
+
+    # this takes a long time to run
+    # @padle.command()
+    # async def filter(self, ctx):
+    #     dbcog = await self.get_dbcog()
+    #     mgraph = dbcog.database.graph
+    #     final = []
+    #     # await ctx.send(mgraph.max_monster_id)
+    #     for i in range(0, 8800): # hard coded bc max_monster_id as calculated above is 15k???
+    #         monster = await dbcog.find_monster(str(i), ctx.author)
+    #         try:
+    #             nextTrans = mgraph.get_next_transform(monster)
+    #             prevTrans = mgraph.get_prev_transform(monster)
+    #             if(monster is not None and monster.name_en is not None and monster.on_na and
+    #                 monster.series.series_type != 'collab' and
+    #                 ((monster.sell_mp >= 50000 and (monster.superawakening_count > 1 or prevTrans is not None)) or
+    #                  "Super Reincarnated" in monster.name_en) and not monster.is_equip and
+    #                     nextTrans is None and monster.level >= 99):
+    #                 final.append(str(i))
+    #         except Exception as e:
+    #             print("Error " + str(i))
+    #             pass
+    #     await ctx.tick()
+    #     print(",".join(final))
+
+    async def ensure_active_game(self, ctx):
```

Comment:
You should make a comment that clarifies that True is the bad answer.  Maybe name it something else so its return type makes more sense?  `is_game_unstartable` or something? idk

Reply:
is_game_unstartable is also not great because "unstartable" has a negative modifier in the word (un). "do_we_forcequit" or something is a totally positively-worded statement where returning True means "yes do a thing" and you can use all "yes do it" language surrounding the code